libcodechecker/analyze/analyzers/analyzer_types.py

```diff
@@ -384,6 +384,20 @@ def __build_clang_tidy_config_handler(args, context):
         # No clang tidy arguments file was given in the command line.
         LOG.debug_analyzer(aerr)
 
+    try:
+        # The config file dumped by clang-tidy contains "..." at the end. This
+        # has to be emitted, otherwise -config flag of clang-tidy can't consume
+        # it.
+        with open(args.tidy_config, 'rb') as tidy_config:
+            lines = tidy_config.readlines()
+            lines = filter(lambda x: x != '...\n', lines)
+            config_handler.checker_config = ''.join(lines)
```

Comment:
maybe `''.join(lines[:-1])` would be enough too if we know that only the last line is not needed?

Reply:
Me too, I was thinking of this. Currently the dump emitted by clang-tidy contains these three dots and an empty line at the end. So actually `lines[:-2]` should be used. This is a somewhat more general solution, if we don't want to rely on the existence of the last empty line. Let's discuss once more which solution should be used.